SpatialGDK/Source/SpatialGDK/Public/Utils/SpatialActorUtils.h

```diff
@@ -44,6 +50,23 @@ inline FString GetConnectionOwningWorkerId(const AActor* Actor)
 	return FString();
 }
 
+template <typename Functor>
+void ForeachReplicatedActorInHierarchy(AActor* HierarchyActor, Functor&& InFunctor)
+{
+	if (HierarchyActor)
+	{
+		if (HierarchyActor->GetIsReplicated())
```

Comment:
If we find a non replicated Actor do we want to continue through its hierarchy?

Reply:
Maybe ? I could imagine a convoluted setup where an actor has some non-replicated actor limbs, which are locally and deterministically animated, but is able to pick up replicated things from the ground.
While I'm not sure it would remain a valid setup for long, I do not want to actually break this scenario.

Reply:
Cool, let's just make the case for everything then 👍